src/Aspire.Dashboard/Model/DashboardClient.cs

```diff
@@ -33,6 +35,9 @@ namespace Aspire.Dashboard.Model;
 internal sealed class DashboardClient : IDashboardClient
 {
     private const string ResourceServiceUrlVariableName = "DOTNET_RESOURCE_SERVICE_ENDPOINT_URL";
+    private const string ResourceServiceDisableAuthVariableName = "DOTNET_RESOURCE_SERVICE_DISABLE_AUTH";
+    private const string ResourceServiceClientCertificatePathVariableName = "DOTNET_RESOURCE_SERVICE_CLIENT_CERTIFICATE_PATH";
```

Comment:
File path won't be enough on its own. There will be people that want to use a cert from the store. I know of security standards where deploying certificates isn't allowed. They must come from the OS store.

Reply:
What ASP.NET Core has for getting certs for endpoints: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel/endpoints?view=aspnetcore-8.0#certificate-sources

* Path and Password to load .pfx files.
* Path, KeyPath and Password to load .pem/.crt and .key files.
* Subject and Store to load from the certificate store.

Reply:
Pushed an update that supports key stores as well. The configuration support for Kestrel doesn't seem applicable to gRPC clients (or at least I couldn't work out how to do it).